pennylane/gradients/__init__.py

```diff
@@ -50,6 +50,16 @@
     hadamard_grad
     stoch_pulse_grad
     pulse_odegen
+    classical_jacobian
+
+Metric tensors
+^^^^^^^^^^^^^^
+
+.. autosummary::
+    :toctree: api
+
+    metric_tensor
+    adjoint_metric_tensor

```

Comment:
```suggestion
    ~pennylane.metric_tensor
    ~pennylane.adjoint_metric_tensor
```

to avoid them showing up in the documentation twice, under two different import names

Reply:
ahhh... you might need to put

```python
.. currentmodule:: pennylane
```

above this autosummary table :(